package.json

```diff
@@ -2,9 +2,10 @@
   "name": "digest",
   "title": "Digest Notifications",
   "description": "Get a daily/weekly digest of what's happening on your site instead of receiving a single email each time.",
-  "version": "2.0.0-alpha",
+  "version": "0.0.0",
```

Comment:
What was the reasoning for this?

Reply:
The package is not released thus it doesn’t require a version.